java/runtime-native/src/main/java/org/ray/spi/impl/DefaultLocalSchedulerClient.java

```diff
@@ -113,6 +118,13 @@ public void reconstructObject(UniqueID objectId, boolean fetchOnly) {
   public void reconstructObjects(List<UniqueID> objectIds, boolean fetchOnly) {
     RayLog.core.info("reconstruct objects {}", objectIds);
     _reconstruct_objects(client, getIdBytes(objectIds), fetchOnly);
+    RayLog.core.info("task id is {}", UniqueIdHelper.computeTaskId(objectIds.get(0)));
```

Comment:
1) this log can be merged with the above one.
2) the message is ambiguous. Imagine when you read `task id is xxx` in tons of log messages. Do you have any idea what this message is about? Thus, as a good practice, we should always be specific about what a log message is about. In this case, it can be something like `Reconstructing objects for task xxx, objects ids are xxx.`.
3) Note that `UniqueIdHelper.computeTaskId` will always be called no matter logging level is below INFO or not. In this case, the computation may not be very heavy. But it's still better to protect the logging call with `if (logger.isInfoEnable())`, to avoid unnecessary computation.